.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# Contributing to VTEX Styleguide
+
+Want to contribute to VTEX Styleguide? There are a few things you need to know.
+
+### Release Schedule
+Thursday release: patch version at the Thursday of every week for routine bug fix (anytime for urgent bug fix).
+
+Tuesday release: minor version at the Tuesday of every week for new features.
+
+Major version releases are not included in this schedule.
+
+### Open Development
+All work on VTEX Styleguide happens directly on GitHub. Both core team members and external contributors send pull requests which go through the same review process.
+
+### Branch Organization
+According to our [release schedule](#release-schedule), we maintain two branches, `master` and `dev`. If you send a bugfix pull request, please do it against the `master` branch, if it's a feature pull request, please do it against the `dev` branch.
```

Comment:
```suggestion
According to our [release schedule](#release-schedule), we maintain two branches: `master` and `dev`. Bug fixes should have `master` as it base branch, while new features should be merged into `dev`.
```

Reply:
Just to be more direct and avoid repetitions

Reply:
done!